docs/getting_started.md

```diff
@@ -306,6 +306,12 @@ CUDA_VISIBLE_DEVICES=0,1,2,3 ./tools/slurm_train.sh ${PARTITION} ${JOB_NAME} con
 CUDA_VISIBLE_DEVICES=4,5,6,7 ./tools/slurm_train.sh ${PARTITION} ${JOB_NAME} config2.py ${WORK_DIR} 4
 ```
 
+## Benchmark
+You can get average training batch for a second, we only care about the model training, not including the IO time and pre-processing time.
```

Comment:
what does this mean: "get average training batch for a second"?

Reply:
average iter/s

Reply:
no, our unit is xxx seconds per iter